.github/workflows/docs.yml

```diff
@@ -41,8 +41,12 @@ jobs:
             "</script>"
 
       - name: cargo doc
-        # (CARGO DOC WITH ALL FEATURES IN THIS FORK)
-        run: cargo doc --locked --all-features --no-deps --package ${{ env.MAIN_CRATE }}
+        # Generate docs in a similar fashion to the docs.rs service
```

Comment:
Wording does not feel 100% right to me. Probably best to leave this for future cleanup (someday). (I have already raised issue #54 to start tracking this kind of cleanup.)